chainio/log.go

```diff
@@ -0,0 +1,29 @@
+package chainio
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var clog btclog.Logger
```

Comment:
nit: any specific reason to rename this from the default `log`? Also, the Godoc comment no longer matches now.

Reply:
yeah initially it was `log`, then I realized it's very easy to use `log` as default while I wanna remember to use `beat.log` since it prints the block height as the header, which has been seen in other packages like `contractcourt`, where the `log` is used instead of `c.log`.